layouts/default.vue

```diff
@@ -24,71 +24,78 @@
             <div v-for="(section, textTag) in group.sections" :key="textTag">
               <v-list-item v-if="section.type === 'external'">
                 <v-list-item-content>
-                  <a :href="section.link" class="ml-2">{{ $t('banner.' + textTag) }}</a>
+                  <a :href="section.link" class="ml-2 black--text">{{ $t('banner.' + textTag) }}</a>
```

Comment:
For each `black--text`, lets replace with `grey--text text--darken-4` (One shade less than black; or darkest shade of grey)? Just to help soften the menu text colours and not be as bold on the eyes.